### DISCUSSION TITLE
Oops! You're using an unsupported method (frontend only). Please serve the WebUI from the backend.

### DISCUSSION THREAD

**Discussion by skhaz** [open-webui/open-webui#6527] [General]
I am trying to serve the application behind Ngrok using Docker Compose. I map port 5173 to 3000, and I see this in the terminal:

chat_app     |   VITE v5.4.6  ready in 2715 ms
chat_app     |
chat_app     |   ➜  Local:   http://localhost:5173/
chat_app     |   ➜  Network: http://192.168.97.3:5173/

However, when I open this URL, the “backend required” warning appears. How can I resolve this?

**Comment by khalilxg** [comment]
same here, how did it resolved?

**Reply by skhaz** [reply]
I use port 8080

**Reply by yo-nagase** [reply]
i've got same issue... are you using docker and change port to 8080?
I'm trying to spin up from code but same error still happening

**Comment by khalilxg** [comment]
So it was browser issue, brave and google throws that error and behave same while mozilla was working fine, it was cache error

**Reply by NUSTHR** [reply]
my case is different. Nothing to do with the browser type. The log showed that service just stuck before the connection with backend service. so wierd

**Reply by rasodu** [reply]
Seems to work. Cleared web browser cache and now it's working.

**Comment by anatolybadanov44-netizen** [comment]
Лечится отключением всяких антизапретов и тп.
